common/grpc/pom.xml

```diff
@@ -55,8 +55,8 @@
             to be able to support 22.2 and 22.3.
         -->
         <dependency>
-            <groupId>org.graalvm.nativeimage</groupId>
-            <artifactId>svm</artifactId>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>graal-sdk</artifactId>
```

Comment:
The GraalVM SDK is already included just under.
Can you remove the duplicated dependency please ?

Reply:
Doh, I forgot to check that in my script, thanks for catching it!